treeherder/model/migrations/0036_remove_bugscacheoccurrence_unique_failureline_bug_occurrence_and_more.py

```diff
@@ -0,0 +1,38 @@
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("model", "0035_bugscache_optional_bugzilla_ref"),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.RemoveConstraint(
+            model_name="bugscacheoccurrence",
+            name="unique_failureline_bug_occurrence",
+        ),
+        migrations.RemoveField(
+            model_name="bugscacheoccurrence",
+            name="failure_line",
+        ),
+        migrations.AddField(
+            model_name="bugscacheoccurrence",
+            name="text_log_error",
```

Comment:
Treeherder stores classifications of jobs (tasks) with bugs (internal issues in the future) which get suggested for failure lines. Based on the discussion last week, I think we should keep that concept.
The unique combination should job + internal issue ID.